docker-compose.yml

```diff
@@ -20,38 +21,51 @@ services:
       - AWS_REGION
     ports:
       - "9090:9090"
+
   catalogs-aviris:
+    image: nasa-hsi-catalogs-aviris
     build:
       context: ./src/catalogs/aviris
     environment:
       - AWS_PROFILE=${AWS_PROFILE:-nasa}
-    image: nasa-hsi-catalogs-aviris
     volumes:
       - ./src/catalogs/aviris:/usr/local/src
       - $HOME/.aws:/root/.aws:ro
+
   activator-aviris-l2:
+    image: activator-aviris-l2
     build:
       context: ./src/pipeline/activator/aviris-l2
     environment:
       - AAL2_FRANKLIN_URL=http://franklin:9090
       - AAL2_S3_BUCKET=aviris-data-dev-${USER}
       - AWS_PROFILE=${AWS_PROFILE:-nasa}
-    image: nasa-hsi-activator-aviris-l2
     volumes:
       - ./src/pipeline/activator/aviris-l2:/usr/local/src
       - ./pipeline-data:/data
+      - /opt/conda/pkgs:/opt/conda/pkgs
```

Comment:
This speeds up local builds when the Docker layer cache is invalidated.

See: https://docs.anaconda.com/anaconda/user-guide/tasks/shared-pkg-cache/